src/beam.ts

```diff
@@ -10,8 +10,44 @@ import { Element } from './element';
 import { Fraction } from './fraction';
 import { Tuplet } from './tuplet';
 import { Stem } from './stem';
+import { Note } from './note';
+import { StemmableNote } from './stemmablenote';
+import { Voice } from './voice';
+import { RenderContext } from './types/common';
+
+export interface BeamInfo {
+  start: number;
+  end?: number;
+}
+
+export interface BeamRenderOptions {
```

Comment:
This can be inline too. No need to export.

Reply:
This one is 11 attributes.

Reply:
Inline?

Reply:
Same -- yes, inline please.